.github/workflows/package.yml

```diff
@@ -247,6 +247,25 @@ jobs:
           GH_TOKEN: ${{ github.token }}
 
       # Test
+      - name: Install Incus
+        if: |
+          startsWith(env.OS, 'almalinux-')
+        run: |
+          # We can use the official Ubuntu APT repository when
+          # ubuntu-latest is Ubuntu 24.04.
```

Comment:
Actually, we can use `incus` on Ubuntu 24:04 beta but there is a following problem now.
So at first, I want to use `ubuntu-latest` now because it's hard for me figure it out.

ref: https://github.com/pgroonga/pgroonga/issues/507#issuecomment-2199634563

Reply:
Could you fix the comment?

Reply:
fix: [41f3ff9](https://github.com/pgroonga/pgroonga/pull/511/commits/41f3ff9a9452f7149c25322e748a953e088be6a7 Sure, I've just updated the comment.